docassemble/juvenilesealing/data/questions/certified-copy-of-dockets-5.yml

```diff
@@ -63,7 +64,7 @@ subquestion: |
 
   ## More about this form
     
-  <span class="new">If you have the information you need,</span>
+  <span style="color: green">If you have the information you need,</span>
```

Comment:
`style="color: green"` was the old way I did this. `class="new"` was the new way. As you can see here, the CSS file didn't change (it should still have the `.new` styles in them. When I opened the CSS file just now, the `.new` class wasn't there. I suspect if I committed now, those rules would show up as changes here.